### ISSUE TITLE
./fritz run --init fails

### ISSUE THREAD

**Issue by dannygoldstein** [fritz-marshal/fritz#10]
on 770f9d6d0

```
(skyportal) Dannys-MacBook-Pro:fritz danny$ ./fritz run --init

Welcome to SkyPortal v0.9.dev0+git20200605.129f1b3 (https://skyportal.io)

cd baselayer && git submodule update --init --remote
docker build -t skyportal/web .
Sending build context to Docker daemon  2.462MB
Step 1/12 : FROM ubuntu:18.04
 ---> ea4c82dcd15a
Step 2/12 : RUN apt-get update &&     apt-get install -y curl build-essential software-properties-common &&     curl -sL https://deb.nodesource.com/setup_12.x | bash - &&     apt-get update &&     apt-get -y upgrade &&     apt-get install -y python3 python3-venv python3-dev                        libpq-dev supervisor                        git nginx nodejs postgresql-client &&     apt-get clean &&     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* &&     useradd --create-home --shell /bin/bash skyportal
 ---> Using cache
 ---> 81b6381c419f
Step 3/12 : RUN python3.6 -m venv /skyportal_env &&         bash -c "source /skyportal_env/bin/activate &&     pip install --upgrade pip"
 ---> Using cache
 ---> fb6f9d31ad57
Step 4/12 : ENV LC_ALL=C.UTF-8
 ---> Using cache
 ---> 633edc1c4820
Step 5/12 : ENV LANG=C.UTF-8
 ---> Using cache
 ---> 2f27ae8c9eeb
Step 6/12 : ADD . /skyportal
 ---> Using cache
 ---> c472611f7a8a
Step 7/12 : WORKDIR /skyportal
 ---> Using cache
 ---> cc2c88329178
Step 8/12 : RUN bash -c "    source /skyportal_env/bin/activate &&         make -C baselayer paths &&     (make -f baselayer/Makefile baselayer dependencies || make -C baselayer dependencies) &&     (make -f baselayer/Makefile baselayer fill_conf_values || make -C baselayer fill_conf_values)"
 ---> Using cache
 ---> 3c45aa466836
Step 9/12 : RUN bash -c "        (make -f baselayer/Makefile bundle || make -c baselayer bundle) &&     rm -rf node_modules &&         chown -R skyportal.skyportal /skyportal_env &&     chown -R skyportal.skyportal /skyportal &&         cp docker.yaml config.yaml"
 ---> Using cache
 ---> 4328d9ce10e7
Step 10/12 : USER skyportal
 ---> Using cache
 ---> 166296661610
Step 11/12 : EXPOSE 5000
 ---> Using cache
 ---> 8968075a56d8
Step 12/12 : CMD bash -c "source /skyportal_env/bin/activate &&              (make log &) &&              make run_production"
 ---> Using cache
 ---> 8e2f64731309
Successfully built 8e2f64731309
Successfully tagged skyportal/web:latest
Creating network "skyportal_default" with the default driver
Creating skyportal_web_1 ... done
Creating skyportal_db_1  ... done

Welcome to SkyPortal v0.9.dev0 (https://skyportal.io)

Checking system dependencies:
[✓] nginx >= 1.7                  [  1.14.0]
[✓] psql >= 9.6                   [   10.12]
[✓] node (npm) >= 5.8.0           [  6.14.4]

[✓] Baselayer installed inside of app
--------------------
[✓] Verifying Python package dependencies
[✓] Silently executing: baselayer/tools/check_js_deps.sh
  Config files: baselayer/config.yaml.defaults
                config.yaml.defaults
                config.yaml
[✓] Connecting to database skyportal
[·] Dropping all tablesDropping tables on database skyportal
[✓] Dropping all tables
[·] Creating tablesCreating tables on database skyportal
Refreshed tables:
 - acls
 - roles
 - role_acls
 - users
 - tokens
 - token_acls
 - user_roles
 - usersocialauths
 - nonces
 - associations
 - codes
 - partials
 - groups
 - group_users
 - streams
 - stream_groups
 - objs
 - filters
 - candidates
 - sources
 - sourceviews
 - telescopes
 - group_telescopes
 - instruments
 - comments
 - photometry
 - spectra
 - thumbnails
 - followuprequests
[✓] Creating tables
    - acls
    - roles
    - role_acls
    - users
    - tokens
    - token_acls
    - user_roles
    - usersocialauths
    - nonces
    - associations
    - codes
    - partials
    - groups
    - group_users
    - streams
    - stream_groups
    - objs
    - filters
    - candidates
    - sources
    - sourceviews
    - telescopes
    - group_telescopes
    - instruments
    - comments
    - photometry
    - spectra
    - thumbnails
    - followuprequests
[✓] Creating permissions
[✓] Creating dummy users
[✓] Creating token
[·] Launching web app & executing API calls
Waiting for server to appear at http://localhost:5000...
App running - continuing with API calls
[✓] Creating dummy groups & filter & adding users
[✓] Creating dummy telescopes & instruments
[✓] Creating dummy candidates & sources
[✓] Launching web app & executing API calls
Traceback (most recent call last):
  File "./fritz", line 318, in <module>
    getattr(sys.modules[__name__], args.command)(args)
  File "./fritz", line 150, in run
    token = generate_skyportal_token()
  File "./fritz", line 64, in generate_skyportal_token
    universal_newlines=True,
  File "/anaconda2/envs/skyportal/lib/python3.6/subprocess.py", line 403, in run
    with Popen(*popenargs, **kwargs) as process:
TypeError: __init__() got an unexpected keyword argument 'capture_output'```

**Comment by dmitryduev** [comment]
This is bc you have python 3.6, but fritz requires 3.7+. I have added a dependency check to fritz, will submit as a part of the upcoming PR.

**Comment by dmitryduev** [comment]
See [here](https://github.com/dmitryduev/fritz/commit/7646c2330eac296c547debf55bf876f09a243f6a)
I should make partial commits a habit :(

**Comment by dmitryduev** [comment]
Addressed in #11
